src/ape/pytest/utils.py

```diff
@@ -10,3 +10,20 @@ class Scope(str, Enum):
 
     def __str__(self) -> str:
         return self.value
```

Comment:
technically, you could change this to an int enum and get relative ordering (e.g. `Scope.CLASS < Scope.FUNCTION`) and then just use `self.name.lower()` here to return the pytest scope when you do `str(scope)`